docs/design/coreclr/profiling/davbr-blog-archive/ELT Hooks - The Basics.md

```diff
@@ -66,15 +66,15 @@ _(Profiler calls this…)_
 ```
   HRESULT SetFunctionIDMapper([in] FunctionIDMapper \*pFunc);
 ```
- 
+
 
      _(Profiler implements this…)_
 ```
 typedef UINT_PTR __stdcall FunctionIDMapper(
                 FunctionID funcId,
                 BOOL *pbHookFunction);
 ```
- 
+
 
 2. When FunctionIDMapper is called:
```

Comment:
This unrelated to the whitespace change, but the first item (Line 8: doesn't appear in GitHub diff, so I can't comment on the exact line) is improperly formatted.

Due to the number of spaces after `1.`, it was treated as a code block, which is completely undesired.

Reply:
Could you please fix in in this PR?